package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "@10up/eslint-config",
-  "version": "1.0.9",
+  "version": "1.1.1-next",
```

Comment:
This will for sure need to be 2.0.0 since it’s a breaking change

Reply:
Yep, will make sure to update to 2.0 before releasing.